QR-dtg.py

```diff
@@ -16,24 +17,36 @@ def clear() -> None:
         pass
         
 def generate_qr() -> None:
-    """Generate a QR code to paste onto a discord nitro template.
-    """
+    #Generate a QR code to paste onto a discord nitro template.
     qr_img = Image.open(os.path.normpath(r"resources/qr_code.png"), "r")
     ovly_img = Image.open(os.path.normpath(r"resources/overlay.png"), "r")
     qr_img.paste(ovly_img, (60, 55))
     qr_img.save(os.path.normpath(r"resources/final_qr.png"), quality=95)
 
 def generate_nitro_template() -> None:
-    """Generate the nitro template using the QR code generated by generate_qr.
-    """
-    nitro_template = Image.open(
-        os.path.normpath(r"resources/template.png"), 
-        "r"
-        )
+    #Generate the nitro template using the QR code generated by generate_qr.
+    nitro_template = Image.open(os.path.normpath(r"resources/template.png"), "r")
```

Comment:
Reducing code line for optimisation.

note: more you reduce code line more the python interpreter is fast;